datasets/conus-reach.data.mdx

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
To be clear, is this file the same as the one added in the other pr https://github.com/NASA-IMPACT/veda-config/pull/189/files ?

Reply:
Yes, it matches with the files I provided to @sahmad3. It should be the same.